### PR TITLE
Cleaned the code and changed the payment method to event brite

### PR DESCRIPTION
**Pull request by christiangoudreau**

### CODE REVIEW

File: src/main/java/com/arcbees/geekette/client/application/ApplicationView.java [hunk 1]
```diff
@@ -70,7 +71,11 @@ protected double interpolate(double progress) {
 
     @UiHandler("buy")
     void onBuy(ClickEvent event) {
-        getUiHandlers().onBuy();
+        Window.open(
+                "http://www.eventbrite.ca/e/billets-bulles-et-geekettes-soiree-de-degustation-de-champagne-11551303259",
+                "_blank", "");
+
+        // getUiHandlers().onBuy();
```
Comment (meriouma) [inline src/main/java/com/arcbees/geekette/client/application/ApplicationView.java:78]: You can remove this I guess

File: src/main/resources/com/arcbees/geekette/client/resources/mediaQueries.css [hunk 2]
```diff
@@ -4,4 +4,4 @@
 
 .noMobile {
     display: none;
-}
\ No newline at end of file
+}
```
Comment (meriouma) [inline src/main/resources/com/arcbees/geekette/client/resources/mediaQueries.css:7]: You can remove this file

File: src/main/java/com/arcbees/geekette/client/application/BuyView.ui.xml [hunk 3]
```diff
@@ -18,6 +18,9 @@
                     <g:TextBox ui:field="email" /><i class="{coreRes.styles.iconBomb}"/>
                 </div>
             </div>
+            <!--<div class="{coreRes.styles.error}">-->
```
Comment (olafleur) [inline src/main/java/com/arcbees/geekette/client/application/BuyView.ui.xml:21]: rm comments

**Comment by olafleur** [comment]
Last commit was more than 6 months ago. Closing. Feel free to reopen a pull request as needed.

**Comment by olafleur** [comment]
Last commit was more than 6 months ago. This pull request should be closed.

**Pull request closed without merging**